src/components/Deployment/Deployment.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState, useEffect } from 'react';
+import styled, { useTheme } from 'styled-components';
+import { AccountStates } from 'etherspot';
+
+// icons
+import { IoMdCheckmark } from 'react-icons/io';
+import { CgSandClock } from 'react-icons/cg';
+
+// components
+import Card from '../Card/Card';
+import { RoundedImage } from '../Image';
+import { useEtherspot } from '../../hooks';
+import Modal from '../Modal/Modal';
+import { Paragraph } from '../Text';
+import { PrimaryButton, SecondaryButton } from '../Button';
+import ErrorMessage from '../Error/ErrorMessage';
+
+// utils
+import { supportedChains } from '../../utils/chain';
+import { deployAccount } from '../../utils/transaction';
+import { Theme } from '../../utils/theme';
+
+const Wrapper = styled.div`
+  margin-top: 10px;
+`;
+
+const Section = styled.div`
+  display: flex;
+  background: ${({ theme }) => theme.color.background.topMenu};
+  margin-top: 5px;
```

Comment:
So I prefer keeping styles at the bottom, but I've noticed you've done it different here and in `ErrorMessage.tsx`, better to choose one way and keep it consistent